docs/components/table.md

```diff
@@ -85,24 +85,169 @@ Lessen the table cell padding
   isSmall />
 ```
 
+### Clickable
+Adds `cursor: pointer` and `user-select: none` styling. 
+
+<KTable :options="$frontmatter.tableOptions" isClickable />
+```vue
+<KTable
+  :options="tableOptions"
+  isClickable />
+```
+
+### Custom Row Attributes
+
+## hasSidebar
+Adds left border to each table row. By default set to true. The colors can be overridden by themes.
+The below example demostrates the disabled state:
+
+<template>
+  <KTable
+    :options="tableOptions"
+    :hasSideBorder="false"
+    />
+</template>
+
+```vue
+<template>
+  <KTable
+    :options="tableOptions"
+    :hasSideBorder="false"
+    />
+</template>
+```
+
```

Comment:
```suggestion
### hasSidebar
Adds left border to each table row. By default set to true. The colors can be overridden by themes.
The below example demonstrates the disabled state:

<template>
  <KTable
    :options="tableOptions"
    :hasSideBorder="false"
    />
</template>

```vue
<template>
  <KTable
    :options="tableOptions"
    :hasSideBorder="false"
    />
</template>
```

## Custom Row Attributes
```

Reply:
can you copy this formatting to https://github.com/Kong/kongponents/pull/237